openpype/hosts/maya/plugins/load/load_image_plane.py

```diff
@@ -13,10 +13,14 @@ def __init__(self, cameras):
         self.setWindowFlags(self.windowFlags() | QtCore.Qt.FramelessWindowHint)
 
         self.camera = None
+        self.static_image_plane = False
+        self.show_in_all_views = False
```

Comment:
Does this mean its off by default in the UI?

Reply:
Exactly.